src/Pass/ImgTagTransformPass.php

```diff
@@ -133,6 +144,19 @@ protected function isSvg(\DOMElement $el)
     }
 
     /**
+     * Detects if the img is a 1x1 pixel. In that case we convert to <amp-pixel> instead of <amp-img>
+     * @param DOMQuery $el
+     * @return bool
+     */
+    protected function isPixel(DOMQuery $el)
+    {
+        if (!$this->setResponsiveImgHeightAndWidth($el)) {
```

Comment:
The generated amp-pixel tag will have `src` as its only attribute because instead of cloning `$el` with `cloneAndRenameDomElement` a new amp-pixel tag is created:

```
$new_dom_el = $dom_el->ownerDocument->createElement('amp-pixel');
$new_dom_el->setAttribute('src', $el->attr('src'));
$dom_el->parentNode->insertBefore($new_dom_el, $dom_el);
```